doc/sphinx/tutorial2.md

```diff
@@ -0,0 +1,81 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
+
+# Tutorial 2
+This tutorial will guide you through using Mir demos to see what Mir is capable
+of, and the usage of Mir libraries to build your own Wayland compositor.
```

Comment:
Diataxis: "A tutorial is a practical activity, in which the student learns by doing something meaningful, towards some achievable goal"

There are two separate goals mentioned here: "using Mir demos to see what Mir is capable of" and "use Mir libraries to build your own Wayland compositor". I don't think they should be combined into one, they are separate lessons targeting distinct objectives.

Reply:
Ok, so as a first step lets split this into two: 
1. "Tutorial: Example Mir Demos"
2. "Tutorial: Using Mir Libraries"
(Naming can be tweaked later)